tools/run-tests.py

```diff
@@ -353,6 +353,7 @@ def run_jerry_tests(options):
 
         if '--profile=es2015-subset' in job.build_args:
             skip_list.append(r"es5.1\/")
+            skip_list.append(r"es2015\/module-imported.js")
```

Comment:
Why is this file is skipped? If the script exports variables/functions, but nobody imports them, is should be also a valid, leak-free test as well.